ee/server/service/mdm.go

```diff
@@ -111,3 +113,34 @@ func (svc *Service) MDMAppleEraseDevice(ctx context.Context, hostID uint) error
 	}
 	return nil
 }
+
+func (svc *Service) MDMAppleEnableFileVaultAndEscrow(ctx context.Context, teamID uint) error {
```

Comment:
So those are the functions I'd call from `fleetctl apply` (and other similar endpoints where we make that change possible) when enabling/disabling disk encryption, right? And then the cron job would pick it up from there.

Reply:
That is right! I put them in `Service` but maybe there's a better place for them? I can move them somewhere else as part of this PR if you have any ideas that will simplify your calling logic.

Reply:
cc: @mna (because I'm not 100% if GitHub will notify you)

Reply:
I think it makes sense to have them there. Of course this means it's not saved as part of the appconfig/team spec transaction, but this is already how many of those things are implemented (e.g. activities). I can detect if the flag changed and call the right service function after the team/appconfig save (and then add the disk encryption activity).

Reply:
thanks, merging!